### DISCUSSION TITLE
`cd` removes current directory from suggestions

### DISCUSSION THREAD

**Discussion by Eragra3** [nushell/nushell#12825] [Q&A]
Hi guys,

I think the following behavior may be expected, but I feel like it's a bit weird.
`cd` seems to be removing current directory from it's autocomplete.
I know it doesn't really do anything to `cd` into current directory, but I think it's nushell trying to be too smart.
It was a bit confusing to me first time I noticed, like I wasn't looking inside correct directory because I don't see what I would expect to see.

![image](https://github.com/nushell/nushell/assets/9082983/e25d251b-36b5-4aea-a924-93b9d3c54b73)

Is this expected? Is it something you got used to or anybody else find this weird?

Love the nushell either way, thanks!
Daniel

**Comment by ghost** [comment]
> Is this expected?

AFAIK it is, but im not familiar enough with the code to check the commit-messages.

> Is it something you got used to

99% of the time i use `..` `cd foo` instead of typing a `/` due to my keyboard layout, but when i do i don't mind it. especially with cd theres no reason to do it and if the parent directory contains 2 directories (current + one) it is a lot quicker.

> anybody else find this weird?

i agree, that this can be disorienting the first time, but since its basically a pure quality of life improvement without a downside i think its a net positive.  
while it usually does not make sense to select the current directory any way except for `.` it would be bad if it was blocked, but just the completion missing is nothing more than a little disorienting (in my opinion).
